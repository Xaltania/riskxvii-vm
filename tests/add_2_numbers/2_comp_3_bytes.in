8388607
3
