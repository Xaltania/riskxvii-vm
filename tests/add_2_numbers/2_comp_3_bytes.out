8388610CPU Halt Requested
